Specs/Scene/ColorMapExpressionSpec.js

```diff
@@ -0,0 +1,121 @@
+/*global defineSuite*/
+defineSuite([
+        'Scene/ColorMapExpression',
+        'Core/Color'
+    ], function(
+        ColorMapExpression,
+        Color) {
+    'use strict';
+
+    function MockStyleEngine() {
+    }
+
+    MockStyleEngine.prototype.makeDirty = function() {
+    };
+
+    function MockFeature(key) {
```

Comment:
In the generic context of a `Feature`, `key` should probably be `value` since it isn't always a key.